README.md

```diff
@@ -54,11 +54,7 @@ Start up a selenium server. By default, the tests expect the selenium server to
 
 Protractor's test suite runs against the included test application.
 
-Install the test application dependencies with:
-
-    npm run install_testapp
-
-Start that up with
+Start the test application with
```

Comment:
This is also important to get started for contributing. The user should install the angular 2 dependencies and serve the Angular app.